csaf_2.0/guidance/faq.md

```diff
@@ -48,6 +48,15 @@ Starting with version CSAF 2.1, [CSAF will support TLP v2](https://github.com/oa
 
 CSAF lister and CSAF aggregator choose on their own which producing parties they add to their lists. Please reach out to the CSAF lister or CSAF aggregator in question. Their contact details are available in the metadata of the list.
 
+### How long should signatures of CSAF documents be valid?
+
+At any given point in time, signatures MUST be valid for 30 more days and SHOULD be valid for at least 90 more days. When signing CSAF documents, the signing party SHOULD comply with or exceed current best practices and guidance on key length.
```

Comment:
I humbly suggest to change to something similar to this for readability:
"At all times, signatures MUST remain valid for a minimum of 30 days and ideally for at least 90 days. When executing CSAF document signatures, the signing party should adhere to or surpass the prevailing best practices and recommendations regarding key length."

Reply:
@santosomar I took your suggestion. However, I'm still wondering about the part "ideally for at least 90 days" as a reader might misunderstand it in a way that you should change signatures every 90 days.